src/modules/tooltip.coffee

```diff
@@ -37,6 +37,8 @@ class Tooltip
     @container.style.left = Tooltip.HIDE_MARGIN
     @quill.setSelection(@range) if @range
     @range = null
+    if dom(@container).hasClass('open')
+      dom(@container).removeClass('open')
```

Comment:
I think `removeClass()` is smart enough to do the right thing if the class wasn't present before, so the `hasClass()` check is a little redundant. 2/10